### PR TITLE
get tv working with v4 of node and newer hapi

### PR DESCRIPTION
**Pull request by lloydbenson**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,43 +14,47 @@
...
     "phantomjs": "1.9.7-15",
-    "sinon": "1.X.X",
-    "sinon-chai": "2.X.X",
-    "watchify": "3.X.X",
-    "hapi": ">=8.X.X"
+    "sinon": "1.x.x",
+    "sinon-chai": "2.x.x",
+    "watchify": "3.x.x",
+    "hapi": "10.x.x",
+    "inert": "3.x.x",
```
Comment (gergoerdosi) [inline package.json]: Can you keep the alphabetical order?

**Comment by lloydbenson** [comment]
@osukaa any issues with this PR?

**Comment by osukaa** [comment]
I'm not able to check it since I'm on another country on vacation :/ I'll be back next Monday 



—

**Comment by lloydbenson** [comment]
Thanks for the update!

**Comment by osukaa** [comment]
Everything seems good to me!

**Comment by lloydbenson** [comment]
Great!  Any plans to merge it?

**Comment by osukaa** [comment]
Yeah, I'm just not sure about versioning.

**Comment by lloydbenson** [comment]
I think a minor version would be appropriate as there aren't any breaking changes, just library updates to make this happen.  As shown by travis this works for both 0.10 and the v4.  I can update this but usually its left up to the maintainer to make this call.

**Comment by osukaa** [comment]
It's merged and published, it closes #119

**Pull request merged**